MN/Algoritmi/Metode-Iterative/Ecuatii-Neliniare/m_bisectiei.m

```diff
@@ -0,0 +1,37 @@
+function [m] = m_bisectiei(f, tol, a, b)
+
+	while 1
+		m = (a + b) / 2;
+		semn = polyval(f, a) * polyval(f, m);
```

Comment:
feval(f, x) == f(x) daca f e un function handler
foloseste feval in loc de polyval